components/rmrk/Profile/ProfileDetail.vue

```diff
@@ -532,6 +544,18 @@ export default class Profile extends mixins(
     })
   }
 
+  get isAllowSetIdentity(): boolean {
+    return !this.displayName && this.isMyProfile && this.hasBlockExplorer
+  }
+
+  get hasBlockExplorer(): boolean {
+    return Object.keys(this.supportedExplorers).includes(this.urlPrefix)
+  }
+
+  get explorer() {
+    return this.supportedExplorers[this.urlPrefix]
+  }
+
```

Comment:
move to utils

Reply:
e, Viki, I am a little confused here. Those are getter that relies on `this.urlPrefix` injected by mixin, so, you mean, move to `PrefixMixin`?

Reply:
```ts
const supportedExplorers: Record<string, string> = {
  rmrk: `[kusama](https://kusama.subscan.io`,
  statemine: `https://statemine.statescan.io`
 }

const hasExplorer = (prefix: string): boolean => {
 return supportedExplorers[prefix] !== undefined
}

const getExlorer = (prefix: string) => {
  return supportedExplorers[prefix]
}
```

then use it in vue.